### PR TITLE
Add business_time:0.13

### PR DESCRIPTION
**Pull request by ksss**
https://github.com/bokmann/business_time

### CODE REVIEW

File: gems/business_time/0.13/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - name: time
+  - name: monitor
+  - name: singleton
+  - name: logger
+  - name: mutex_m
+  - name: minitest
+  - name: json
+  - name: benchmark
+  - name: digest
+  - name: set
```
Comment (pocke) [inline gems/business_time/0.13/manifest.yaml:12]: I guess most of all dependencies are activesupport's. If so, they are unnecessary here.

Comment (ksss) [reply gems/business_time/0.13/manifest.yaml]: I removed it because the dependency on activesupport alone will provide everything.

**Comment by ksss** [comment]
I rebased by current main and add enable "untyped call" checking.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -12,6 +12,14 @@
...
 /gems/rails-dom-testing @ksss
 /gems/yard              @ksss
+/gems/ast            @pocke
+/gems/aws-sdk-*      @ksss
+/gems/business_time  @ksss
+/gems/dogstatsd-ruby @VTRyo
+/gems/faraday        @yykamei
+/gems/i18n           @takahashim
+/gems/rolify         @ksss
+/gems/stackprof      @pocke
```
Comment (pocke) [inline .github/CODEOWNERS:22]: Could you fix this file to avoid duplication?

Comment (ksss) [reply .github/CODEOWNERS]: fixed 👍

File: gems/business_time/0.13/manifest.yaml (hunk 1 above)
Comment (ksss) [reply gems/business_time/0.13/manifest.yaml]: Only `set` was left.

**Comment by ksss** [comment]
Updated all!

**Review by pocke — APPROVED** [review]
Thanks!

**Pull request merged**